cypress/e2e/configuration/shippingMethods/shippingWeights/weightRecalculate.js

```diff
@@ -54,10 +54,12 @@ describe("As a staff user I want to change shop default weight unit", () => {
       });
   });
 
-  it(
+  xit(
```

Comment:
```suggestion
  it.skip(
```
I think it  will be more readable that we skip the test and it is not a typo.